src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -749,10 +750,10 @@ def forward(
         Examples:
 
         ```python
-        >>> from transformers import LayoutLMTokenizer, LayoutLMModel
+        >>> from transformers import AutoTokenizer, LayoutLMModel
```

Comment:
I see you update here to `AutoTokenizer`, but can we stay consistent and update all code examples in that case?

Reply:
Good catch. This was a pre-existing bug in the docstring (the code used `AutoTokenizer`). But for consistency's sake, I have changed the import back to `LayoutLMTokenizer` and fixed the code to use it.

Reply:
Actually it ends up making a difference to use `AutoTokenizer` for the question answering one, so I've changed all of them to AutoTokenizer.

Reply:
Great, can you also remove `_TOKENIZER_FOR_DOC = "LayoutLMTokenizer"` in modeling_layoutlm.py? As it's not being used anywhere

Reply:
done

Reply:
Ok